src/browser-polyfill.js

```diff
@@ -446,7 +448,23 @@ if (typeof browser === "undefined") {
 
     const wrappedSendMessageCallback = ({reject, resolve}, reply) => {
       if (chrome.runtime.lastError) {
-        reject(chrome.runtime.lastError);
+        // Passing a callback to `chrome.runtime.sendMessage`
```

Comment:
Let's trim this inline comment a bit, and we should actually mention that this is needed to match the behavior on Firefox, e.g. something like the following could be enough:

```js
// Detect when none of the listener replied to the sendMessage call and resolve
// the promise to undefined as in Firefox.
// See https://github.com/mozilla/webextension-polyfill/issues/130        
```